README.adoc

```diff
@@ -90,13 +92,27 @@ Creating pact-broker_postgres_1 ... done
 Creating pact-broker_pact-broker_1 ... done
 ```
 
-Go to the http://localhost:9292/[^] URL to confirm that you can access the user interface (UI) of the Pact Broker, as shown in the following image:
+// Static guide instruction
+ifndef::cloud-hosted[]
+Go to the http://localhost:9292/[^] URL to confirm that you can access the user interface (UI) of the Pact Broker.
+endif::[]
 
-image::pact-broker-webpage.png[Pact Broker webpage, align="left"]
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Confirm the Pact Broker is working.
+Select **Launch Application** from the menu of the IDE and type **9292** to specify the port number for the Pact Broker service. 
+Click the **OK** button. 
+The Pact Broker can also be found at the **`https://accountname-9292.theiadocker-4.proxy.cognitiveclass.ai`** URL, 
+where **accountname** is your account name.
+endif::[]
 
-{empty} +
+Confirm you can you use the user interface of the Pact Broker, as shown in the following image:
```

Comment:
- [ ] "Confirm you can you use the user interface of the Pact Broker" looks not right
- [ ] resume all `{empty} +`, so that more space after the images

Reply:
Rewrote sentence and put back `{empty} +` after images